src/pages/_app.tsx

```diff
@@ -4,15 +4,18 @@ import { SessionProvider } from "next-auth/react";
 import { api } from "../utils/api";
 
 import "../styles/globals.css";
+import { ThemeProvider } from "next-themes";
 
 const MyApp: AppType<{ session: Session | null }> = ({
   Component,
   pageProps: { session, ...pageProps },
 }) => {
   return (
-    <SessionProvider session={session}>
-      <Component {...pageProps} />
-    </SessionProvider>
+    <ThemeProvider defaultTheme="light">
```

Comment:
Set the default theme to dark. Let's not burn the user's eye with the bright light theme if they open the site at night.

```suggestion
    <ThemeProvider defaultTheme="dark">
```